desk.go

```diff
@@ -201,6 +201,22 @@ func (l *deskLayout) scaleVars(scale float32) []string {
 	}
 }
 
+// MouseInNotify can be called by the window manager to alert the desktop that the cursor has entered the canvas
+func (l *deskLayout) MouseInNotify(x int, y int) {
+	barX, barY := appBar.Position().X, appBar.Position().Y
+	barWidth, barHeight := appBar.Size().Width, appBar.Size().Height
+	if x >= barX && x <= barX+barWidth {
+		if y >= barY && y <= barY+barHeight {
+			appBar.MouseIn(nil)
```

Comment:
This seems dangerous - the contract for MouseIn is that it takes a *fyne.MouseEvent, so why not construct one with the information we have? there is no button or modifier so we can just pass the position and absolute position using the position passed into this function (when updated)?